crates/orchestrator/src/alerts/mod.rs

```diff
@@ -7,5 +7,5 @@ pub mod aws_sns;
 #[async_trait]
 pub trait Alerts: Send + Sync {
     /// To send an alert message to our alert service
-    async fn send_alert_message(&self, message_body: String) -> color_eyre::Result<()>;
+    async fn send_alert_message(&self, message_body: String) -> color_eyre::Result<Option<String>>;
```

Comment:
This was done since the `publish.message_id` is optional, thus leading to a possibility that if the message doesn't come back we do return None, 
Questions : 
- Initially this method doesn't return anything, why are we fine with that ?
- Do we want to return an error if the `message_id` field is return as `None` ?

Reply:
> Initially this method doesn't return anything, why are we fine with that ?

we shouldn't be if None means that it wasn't sent

> Do we want to return an error if the message_id field is return as None ?

what does None mean?